authnz/src/main/java/io/stargate/auth/TypedKeyValue.java

```diff
@@ -0,0 +1,64 @@
+package io.stargate.auth;
+
+import io.stargate.db.datastore.query.Value;
+import io.stargate.db.datastore.query.WhereCondition;
+
+public class TypedKeyValue {
+
+  String name;
+  String type;
+  Object value;
```

Comment:
`private`?

Reply:
Also, are setters used? Could these fields also be `final`?

Reply:
Right, forgot to circle back on the `private`, thanks. Probably not so I can make them `final`.